target/linux/lantiq/base-files/etc/board.d/02_network

```diff
@@ -122,6 +122,8 @@ BTHOMEHUBV2B|BTHOMEHUBV3A)
 	lan_mac=$(mtd_get_mac_ascii uboot_env ethaddr)
 	wan_mac=$(macaddr_add "$lan_mac" 1)
 	ucidef_set_interface_lan 'eth0'
```

Comment:
I might be wrong, but he ucidef_set_interface_lan should be removed as soon as you have a vlan setup.